<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OnDA Crystallography</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout0">
      <item>
       <widget class="QSplitter" name="splitter0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="ImageView" name="imageView"/>
        <widget class="QSplitter" name="splitter1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="PlotWidget" name="hitRatePlotWidget"/>
         <widget class="PlotWidget" name="saturationPlotViewer"/>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout0">
        <item>
         <widget class="QPushButton" name="resetPeaksButton">
          <property name="text">
           <string>Reset Peaks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetPlotsButton">
          <property name="text">
           <string>Reset Plots</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="delayLabel">
          <property name="text">
           <string>Estimated Delay: -</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="resolutionRingsCheckBox">
          <property name="text">
           <string>Resolution Rings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="resolutionRingsLineEdit"/>
        </item>
        <item>
         <widget class="QCheckBox" name="accumulatedPeaksCheckBox">
          <property name="text">
           <string>Show Accumulated Peaks</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
